lib/sportradar/api/nfl/stat/extra_points.rb

```diff
@@ -0,0 +1,50 @@
+module Sportradar
+  module Api
+    class Nfl::Stat::ExtraPoints < Nfl::StatPack
+      attr_accessor :attempts, :made, :blocked, :pass_attempts, :pass_successes, :rush_attempts, :rush_successes, :defense_attempts, :defense_successes, :turnover_successes
+
+      def set_stats
+        kick_data = response['kicks'] || response
+        @attempts = kick_response["attempts"]
+        @made     = kick_response["made"]
+        @blocked  = kick_response["blocked"]
+        if response['conversions']
+          @pass_attempts      = response["pass_attempts"]
+          @pass_successes     = response["pass_successes"]
+          @rush_attempts      = response["rush_attempts"]
+          @rush_successes     = response["rush_successes"]
+          @defense_attempts   = response["defense_attempts"]
+          @defense_successes  = response["defense_successes"]
+          @turnover_successes = response["turnover_successes"]
+        end
+      end
+
+    end
+  end
+end
+
+extra_points = 
```

Comment:
I meant to leave it commented out. There's two stat types that have differently formatted data, and it seemed like leaving a copy for future reference might be nice.